parlai/tasks/eli5/agents.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from parlai.core.teachers import FixedDialogTeacher
+from .build import build
+import os
+import json
+
+
+class ELI5Teacher(FixedDialogTeacher):
+    """
+    ELI5 Teacher, taken from https://github.com/facebookresearch/ELI5.
+    """
+
+    def __init__(self, opt, shared=None):
+        super().__init__(opt, shared)
+        build(opt)
+        self.id = 'eli5'
+        self.messages = self.load_eli5(opt)
+        self.reset()
+
+    @staticmethod
+    def add_cmdline_args(argparser):
+        group = argparser.add_argument_group('ELI5 Knowledge arguments')
+        group.add_argument(
+            '--knowledge',
+            type='bool',
+            default=True,
+            help='Whether to include supporting document knowledge',
+        )
+
+    def load_eli5(self, opt):
+        """
+        Load data based on data split.
+        """
+        dp = opt['datapath']
+        dt = opt['datatype'].split(':')[0]
+        eli_path = "eli5/processed_data/selected_15_1/explainlikeimfive_"
```

Comment:
This needs a a try/except path, that will raise an error when the user hasn't run the downloading instructions completely.

Reply:
And by try except path, I mean, we should loudly error and point the user to the docs.

Reply:
Should throw a FileNotFoundError and tell them to follow the instructions now.

Reply:
That works for me.